libs/core/langchain_core/load/dump.py

```diff
@@ -6,7 +6,18 @@
 
 def default(obj: Any) -> Any:
     """Return a default value for a Serializable object or
-    a SerializedNotImplemented object."""
+    a SerializedNotImplemented object.
+
+    Args:
+        obj: The object to serialize to json if it is a Serializable object.
+
+    Returns:
+        A json serializable object.
+
+    Raises:
+        ValueError: If the object is not a Serializable object or
```

Comment:
i actually dont think this raises any errors on a not Serializable object

Reply:
Thanks! Good catch.